weekly-report/main.py

```diff
@@ -103,32 +127,117 @@ def handler(event=None, context=None):
     sig_chart = get_significance_bar()
     mag_map = get_magnitude_map()
     state_map = get_state_risk_map()
+    two_layer_pie = create_two_layer_pie()
 
     mag_html = convert_altair_chart_to_html_embed(mag_map)
     state_html = convert_altair_chart_to_html_embed(state_map)
+    pie_html = convert_altair_chart_to_html_embed(two_layer_pie)
+
+    total_earthquakes = get_total_number_earthquakes(con)
+    highest_magnitude = get_top_magnitude_earthquake(con)[1]
+    most_significant_earthquake = get_top_significant_earthquakes(con)[0][1]
     logging.info("Altair charts converted to html")
 
+    prev_monday = previous_monday(CURRENT_DATE).strftime("%B %d, %Y")
+
     html_report = f"""
```

Comment:
Good job on the html, I really like the detail you have included and the formatting you have done